sugoi-api-ldap-config-provider/src/main/java/fr/insee/sugoi/config/LdapRealmProviderDAOImpl.java

```diff
@@ -336,8 +224,66 @@ public ProviderResponse deleteRealm(String realmName, ProviderRequest providerRe
       response.setEntityId(realmName);
       return response;
     } catch (LDAPException e) {
-      throw new RuntimeException("Failed to delete realm " + realmName, e);
+      throw new RealmWriteFailureException("Failed to delete realm " + realmName, e);
+    }
+  }
+
+  private List<UserStorage> loadUserStorages(SearchResultEntry realmEntry) throws LDAPException {
+    String baseName = realmEntry.getAttribute("cn").getValue();
```

Comment:
use 
  @Value("${fr.insee.sugoi.config.ldap.profils.pattern:cn=Profil_{realm}_WebServiceLdap}")
  private String realmEntryPattern;
instead of hardcoded "cn"